Samples/Client/SwiftClient/SwiftClientTests/SwiftClientTests.swift

```diff
@@ -0,0 +1,270 @@
+import XCTest
+@testable import SwiftClient
+import WebRTC
+
+class SwiftClientTests: XCTestCase {
+    var videoStreamVC: VideoStreamViewController!
+    var timeoutSecs: TimeInterval! = TimeInterval(50)
+    let dispatchGroup = DispatchGroup() //Force only one test to run at a time
+    let randomPeerId = 10000
+    
+    override func setUp() {
+        super.setUp()
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        videoStreamVC = storyboard.instantiateViewController(withIdentifier: "VideoStreamViewController") as! VideoStreamViewController
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        videoStreamVC.disconnectAsync(completionHandler: nil)
+        videoStreamVC = nil
+        timeoutSecs = nil
+    }
+    
+    func testSignInAsync() {
+        dispatchGroup.enter()
+        let signInExpectation = expectation(description: "signInExpectation")
+        self.videoStreamVC.signInAsync { (data, response, error) in
+            XCTAssertNil(error)
+            XCTAssertNotNil(data)
+            XCTAssertNotNil(response)
+            XCTAssertEqual(response!.statusCode, 200)
+            self.videoStreamVC.myId = 1000 // populate to avoid crashing from not calling connectAsync
```

Comment:
can you explain further why things crash without this? (at least here in the PR comments)

Reply:
This existed because myId would be populated in the connectAsync function and at this time the connectAsync function had yet to be called. I am getting rid of this and just making myId initialize to -1 when the app starts (the disconnected state, which makes more sense than leaving it nil and populating it in connectAsync)